editor/src/components/canvas/canvas-strategies/post-action-options/post-action-paste.ts

```diff
@@ -592,7 +592,9 @@ function pasteToReplaceCommands(
   originalMetadata: ElementInstanceMetadataMap,
   originalPathTree: ElementPathTrees,
 ): Array<CanvasCommand> {
-  const pasteCommands = targets.flatMap((target) => {
+  const targetsWithoutRoot = targets.filter((target) => !EP.isRootElementOfInstance(target))
```

Comment:
It might be safer here to rename the function's input `targets` input param to `unfilteredTargets`. I'm thinking that if someone comes in here later and generates another type of command for each of the targets, the naming of that param would signify that they probably don't want to blindly use it (especially since such a change would be unlikely to show this filtering line in a PR diff)